unit-tests/run-unit-tests.py

```diff
@@ -467,7 +467,6 @@ def test_wrapper( test, configuration=None, repetition=1, serial_numbers=None ):
         if test_wrapper_( test, configuration, repetition, retry, test.config.retries, serial_numbers ):
             return True
 
-    log._n_errors += 1
```

Comment:
Will it still count errors w/o this change?

Reply:
Yes, errors are counter by the failed test calling `log.e`.
In this function there was a loop for the retries that "knew" that each retry will increase the log, so it decreased it back, and this line was supposed to add one to the counter for all retries. Previous PR removed the decrease (and made the `log.e` call only on last retry), but I did not remove this increase.